tests/Fixer/ClassNotation/VisibilityRequiredFixerTest.php

```diff
@@ -870,6 +870,16 @@ class Foo
                 }
                 PHP,
         ];
+
+        yield 'abstract property hooks' => [
+            <<<'PHP'
+                <?php
+                abstract class Foo
+                {
+                    abstract protected string $bar { get => "a"; set; }
```

Comment:
already added and fixed
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/tests/Fixer/ClassNotation/VisibilityRequiredFixerTest.php#L888